docs/sagemaker/train.md

```diff
@@ -99,6 +99,12 @@ _Note that SageMaker doesn’t support argparse actions. For example, if you wan
 
 Look [here](https://github.com/huggingface/notebooks/blob/master/sagemaker/01_getting_started_pytorch/scripts/train.py) for a complete example of a 🤗 Transformers training script.
 
+## Training Output Management
+
+`output_dir` defaults to '/opt/ml/model' which saves both checkpoints and models to '/opt/ml/model', and results in a large output `tar.gz` containing all checkpoint files and model. A large artifacts most of them possibly useless for deployment.
```

Comment:
```suggestion
If `output_dir` in the `TrainingArguments` is set to '/opt/ml/model' the Trainer saves all training artifacts, including logs, checkpoints, and models. Amazon SageMaker archives the whole '/opt/ml/model' directory as `model.tar.gz` and uploads it at the end of the training job to Amazon S3. Depending on your Hyperparameters and `TrainingArguments` this could lead to a large artifact (> 5GB), which can slow down deployment for Amazon SageMaker Inference. 
```

Reply:
why did do resolve this comment?

Reply:
Sorry, I have overlooked it and clicked by mistake.